pkg/docker/docker.js

```diff
@@ -108,6 +108,28 @@ define([
         return promise;
     };
 
+    docker.check_image = function check_image(image) {
```

Comment:
Could we name this function docker.inspect_image()? That would match the command line usage, and be more easily readable.